packages/contract/src/contract-call-script/src/main.sw

```diff
@@ -1,34 +1,36 @@
 script;
 
-use std::context::registers::*;
+use std::contract_id::ContractId;
 
-const CONTRACT_ID_LEN = 32;
-const WORD_SIZE = 8;
+fn get_script_data<T>() -> T {
```

Comment:
This is actually quite an interesting workaround for not having script `main()` parameters. 👍

Reply:
Yeah, I love it! Hopefully will be in std soon: https://github.com/FuelLabs/sway/issues/1062

BUT, being able to have `main()` args and also forc generating JSON ABIs from scripts would be great.